include/gmapping/gridfastslam/gridslamprocessor.h

```diff
@@ -305,11 +308,10 @@ namespace GMapping {
     // stream in which to write the messages
     std::ostream& m_infoStream;
     
-    
     // the functions below performs side effect on the internal structure,
     //should be called only inside the processScan method
   private:
-    
+    std::vector<double> durations;
```

Comment:
Why not just use two members such as  sumscore and update_count to compute the average  execution time, since  the execution time of each function call is of little use?